react-native-cli/index.js

```diff
@@ -240,7 +240,9 @@ function createProject(name, options) {
     version: '0.0.1',
     private: true,
     scripts: {
-      start: 'node node_modules/react-native/local-cli/cli.js start'
+      start: 'node node_modules/react-native/local-cli/cli.js start',
+      ios: 'react-native run-ios',
```

Comment:
Are you sure this is actually going to work? In this context, `react-native` points to the executable in `node_modules/.bin/react-native`, which is this: https://github.com/facebook/react-native/blob/master/package.json#L114 and is used to warn users that `react-native` cannot be installed globally. There's imo no way to refer to `react-native-cli` from npm scripts context as far as I remember.

If you want to refer to the local-cli, access it like in `start`.